lib/bolt_server/transport_app.rb

```diff
@@ -53,8 +63,8 @@ def initialize(config)
       content_type :json
 
       body = JSON.parse(request.body.read)
-      schema_error = JSON::Validator.fully_validate(@schemas["ssh-run_task"], body)
-      return [400, schema_error.join] if schema_error.any?
+      error = validate_schema(@schemas["ssh-run_task"], body)
+      return [400, JSON.generate(error)] unless error.nil?
```

Comment:
I think this return has to be outside of the helper in order to return from the sinatra endpoint instead of the helper function. I could be wrong though - lemme know if there's a better way to do this.